docs/static/monitoring/monitoring-ea-dashboards.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[dashboard-monitoring-with-elastic-agent]]
+=== Collect {ls} monitoring data for dashboards
+[subs="attributes"]
+++++
+<titleabbrev>Collect monitoring data for dashboards</titleabbrev>
+++++
+
+ToDo: Add brief intro describing the feature and value prop. 
+So far, we haven't differentiated between the two. 
+Add info to help them decide which one they need.  
+
+**Prerequisites**
+
+Complete these steps as you prepare to collect and ship monitoring data for dashboards:
+
+[[disable-default-db]]
+.Disable default collection of {ls} monitoring metrics
+[%collapsible]
+====
+include::monitoring-prereq-disable-default.asciidoc[]
+====
+
+[[define-cluster__uuid-db]]
+.Specify the target cluster_uuid (optional)
+[%collapsible]
+====
+include::monitoring-prereq-define-cluster.asciidoc[]
+====
+
```

Comment:
We (unfortunately) still need to create a user when the standalone agent is being used - we can either move that down into the standalone widget, create a further widget here, or add this block here:

```suggestion
[[create-user-db]]
.Create a monitoring user (standalone agent only)
[%collapsible]
====
include::monitoring-prereq-create-user.asciidoc[]
====
```

I'm ok with adding this block - I think we have done a decent enough job of organizing the rest of the content, that an optional block here may be ok, despite it being additional cognitive load for fleet users